stix_shifter.py

```diff
@@ -2,7 +2,7 @@
 import sys
 import importlib
 
-MODULES = ['qradar', 'dummy']
+MODULES = ['qradar', 'dummy', 'aql_passthrough']
```

Comment:
this PR contains the same change: [https://github.com/IBM/stix-shifter/pull/9](url)
Should I ask Ben to discard his change?